students/olszewski_bartosz/lesson_02_flow_control/knight_move.py

```diff
@@ -0,0 +1,10 @@
+base_x = int(input())
+base_y = int(input())
+direct_x = int(input())
+direct_y = int(input())
+if (abs(direct_x - base_x) == 2) and (abs(direct_y - base_y) == 1):
+    print('YES')
+elif (abs(direct_x - base_x) == 1) and (abs(direct_y - base_y) == 2):
```

Comment:
It might be compressed to one if  statement
if (abs(direct_x - base_x) == 2) and (abs(direct_y - base_y) == 1) or \
    (abs(direct_x - base_x) == 1) and (abs(direct_y - base_y) == 2):
    print('YES')